pandas/tests/plotting/test_frame.py

```diff
@@ -144,6 +144,12 @@ def test_plot(self):
         result = ax.axes
         assert result is axes[0]
 
+    # GH 25587
+    def test_integer_array_plot(self):
+        s = Series([4, 5, 3, 2], dtype="UInt32")
```

Comment:
Can we check with other EA integer dtypes?

(i.e. parameterize over `dtype` to test all possible EA integer types)

Reply:
That feels like overkill, especially since these tests will probably take a measurable amount of time. I'm not too worried about differences between the various integer dtypes. We can address those if they come up.